backend/.sqlx/query-14af4de80324b022defcc022ae84a4956cb9189f2baed776e733f138fb7ccb3e.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
what are these?

Reply:
sqlx files for rust build to work with docker (will be useful for building in ci and having cd)